package.json

```diff
@@ -27,7 +27,6 @@
     "@angular/platform-browser-dynamic": "~10.0.11",
     "@angular/router": "~10.0.11",
     "@stoplight/elements-web-components": "^6.0.0-beta.166",
-    "rxjs": "~6.6.2",
```

Comment:
Isn't rxjs used by angular itself?

Reply:
It is, but it's not a peer dependency, so doesn't matter.
![image](https://user-images.githubusercontent.com/543372/96255920-078cd000-0fb8-11eb-808f-932b2b6a82ba.png)